### DISCUSSION TITLE
[Question] is CRC available on linux arm64 based CPUs ?

### DISCUSSION THREAD

**Discussion by ayoub-tl** [crc-org/crc#3776] [Q&A]
is CRC available on linux arm64 based CPUs ? and is this feauters planned ?
thank you

**Comment by praveenkumar** [comment]
> is CRC available on linux arm64 based CPUs ? and is this feauters planned ?

What is the usecase? As of now CRC is for running openshift cluster locally on developer's laptop/desktop and arm64 on linux is not as common for laptop/desktop. We deliver bundles for mac-M1 which is arm64 based.

**Comment by gbraad** [comment]
Not planned. If you want to run on an Ampere Altra, please consider using the openshift-installer for now. Images are possible to run on this target, though the main executable has not been built with this in mind at the moment.

> arm64 on linux is not as common for laptop/desktop.

There are some boards, like the Solidrun Honeycomb. Though availability is still an issue. Be aware, a RasPi just won't do it for OCP/OKD... only Microshift...

**Comment by cfergeau** [comment]
Had a similar request today during. A team is using crc to test their application/product on an openshift/microshift cluster, and they need their testing to cover both amd64 and arm64.

**Comment by cgoguyer** [comment]
I'm also interested in crc on arm linux desktop, any updates on this topic or definitely abandoned?

**Reply by cfergeau** [reply]
There was some progress on this, see https://github.com/crc-org/crc/wiki/Using-crc-on-arm64-machine (the version numbers need to be adjusted)
